Add-ons/UmbracoForms/Editor/Creating-a-Form/Conditional-Logic/index.md

```diff
@@ -1,58 +1,55 @@
 # Setting up conditional logic on fields
 
-Only showing a field if the value of another field is ... That's what you can do with conditional logic.
+Some times you might have a field in your form, that you only want to show if the user has entered a specific value in another field: 
 
-## Example form
-Take a look at the following as an example
+*Only show this field if the value of that field is x*
 
-![Example form](ExampleForm.png)
+You can achieve this by setting up **conditional logic**.
 
-It make sense to only show the email/phone field when that corresponds with the option chosen in the 'how should we contact you?' field.
+## Example
 
-## Enabling conditional logic
+Take a look at the following as an example
 
-Enabling conditions for a field can be done on the additional settings of a field.
+![Example form](images/ExampleForm.png)
 
+In this case it would make sense to **only show the email/phone field when that corresponds with the option chosen in the 'How should we contact you?' field**.
 
-![Enable conditions](EnableConditions.png)
+The achieve that, we need to enabling conditions for the **Email** and **Phone** fields. This can be done in the additional settings.
 
-Simply check the enable conditions box and you should then see more options
+![Enable conditions](images/EnableConditions.png)
 
-![Conditions](Conditions.png)
+Enabling conditions will give you more options:
 
-### Action type
+![Conditions](images/conditions.png)
 
-First thing that needs to be chosen is the action type this can be either show or hide
+### Action and Logic types
 
-![ActionType](ActionType.png)
+First thing that needs to be chosen is the **action type**:
 
 - Show: the field will be hidden unless the rules match
 - Hide: the field will be hidden if the rules match
 
-### Logic type
-
-Then you'll need to specify if all rules need to match. This setting is only important if you have multiple rules.
+Next up, you'll need to specify the **logic type**. This setting is only important if you have multple rules.
 
-![LogicType](LogicType.png)
-
-- All: all of the rules must match
-- Any: any of the rules may match
+- All: All of the rules must match
+- Any: Any of the rules may match
 
 ## Adding a new condition
 
 When adding a new condition you'll need to select the field where you want to evaluate the value and then you can also select an operator.
 
-In this example I only want to show the email field if the value of the 'how should we contact you' field is 'Email'
+In this example we only want to show the 'Email' field if the value of the 'How should we contact you' field is 'Email'. In the same way we only want to show the 'Phone' field, if the value of the 'How should we contact you' field is 'Phone'.
```

Comment:
There's __a lot__ of apostrophes here, maybe make some of these fields inline code or bold or italic or something? 😁